first_diff.py

```diff
@@ -1,255 +1,55 @@
 #!/usr/bin/env python3
 
-import os.path
+from __future__ import annotations
+
 import argparse
-from subprocess import check_call
-
-parser = argparse.ArgumentParser(
-    description="Find the first difference(s) between the built ROM and the base ROM."
-)
-parser.add_argument(
-    "-c",
-    "--count",
-    type=int,
-    default=5,
-    help="find up to this many instruction difference(s)",
-)
-parser.add_argument(
-    "-d",
-    "--diff",
-    dest="diff_args",
-    nargs="?",
-    action="store",
-    default=False,
-    const="prompt",
-    help="run diff.py on the result with the provided arguments"
-)
-parser.add_argument(
-    "-m", "--make", help="run make before finding difference(s)", action="store_true"
-)
-args = parser.parse_args()
-
-diff_count = args.count
-
-if args.make:
-    check_call(["make", "-j4", "COMPARE=0"])
-
-baseimg = f"baserom.z64"
-basemap = f"expected/build/z64.map"
-
-myimg = f"zelda_ocarina_mq_dbg.z64"
-mymap = f"build/z64.map"
-
-if not os.path.isfile(baseimg):
-    print(f"{baseimg} must exist.")
-    exit(1)
-if not os.path.isfile(myimg) or not os.path.isfile(mymap):
-    print(f"{myimg} and {mymap} must exist.")
+from pathlib import Path
+import rabbitizer
```

Comment:
This would be nice also wrapped in a try except ImportError, rabbitizer isn't a dependency for building oot